src/schema/v2/artwork/collectorSignals.ts

```diff
@@ -3,8 +3,6 @@ import { GraphQLInt, GraphQLObjectType } from "graphql"
 import { ResolverContext } from "types/graphql"
 import { PartnerOfferToCollectorType } from "../partnerOfferToCollector"
 import { isFeatureFlagEnabled } from "lib/featureFlags"
-import { some } from "lodash"
-import { isFieldRequested } from "lib/isFieldRequested"
 
 const NOW = new Date()
```

Comment:
Does this get re-evaluated or is it 'fixed' at the time that Metaphysics boots and initially loads this file?